articles/how-to-use-local-storage-with-JavaScript/index.md

```diff
@@ -0,0 +1,284 @@
+
+For example, to retrieve the above `Car` object, we will use the following statement.
+
+```JavaScript
+Window.localStorage.getItem('car');
+```
+
+The above statement will return something like this;
+
+```JavaScript
+"{brand:"Suzuki",color:"white",price:"10000"}"
+```
+
+You should convert it to an object using `JSON.parse()` to use it in your code.
+
+```JavaScript
+JSON.parse(window.localStorage.getItem('car'));
+```
+
+#### removeItem()
+This method is used to delete an item from the local storage. The `removeItem()` method requires a key as a parameter.
+
+```javascript
+window.localStorage.removeItem('brand');
+```
+
+#### clear()
+This method is used to clear all values stored in the local storage. It does not require any parameters.
+
+```JavaScript
+Window.localStorage.clear()
+```
+
+### Project
+Now that we have learned about the primary functions of local storage, let's create a web application that stores, retrieves, deletes, and clears items from the local storage.
+
+Create a new folder and open it in your code editor. Create two files, `index.html` and `main.js`. The `index.html` will showcase the web page to the user, while the `main.js` will store our javascript functions. These functions will be used to access different functionalities of the `Local Storage`.
+
+### Let's Code
+Our `index.html` will have a `form` and several `buttons`, as shown below.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>localStorage</title>
+  <link rel="stylesheet" href="./css/bootstrap.css">
+  <script type="text/javascript" src="main.js"></script>
+
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
+        aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarCollapse">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <br>
+  <br>
+  <div class="container" id="formDiv">
+        <form class="form-signin" id="carForm">
+            <h1>Please sign in</h1>
+            <label for="carBrand" class="sr-only">Car</label>
+            <input type="text" id="carBrand" class="form-control" placeholder="Car brand" required autofocus><br>
+            <label for="carPrice" class="sr-only">Price</label>
+            <input type="text" id="carPrice" class="form-control" placeholder="Price" required><br>
+            <button class="btn btn-lg btn-primary btn-block" type="submit">Submit</button>
```

Comment:
Update the button text to Store Records. To be consistent with the other buttons.